### ISSUE TITLE
Proposal: rename mark to tag

### ISSUE THREAD

**Issue by nat-418** [cbochs/portal.nvim#1]
The term "mark" has special meaning in Vim / Neovim, and it's confusing to re-use it to mean something else. The term "tag", while also used for ctags, is well-understood to be generic. Renaming marks  to tags would reduce confusion about what is happening with your plugin, in my opinion.

**Comment by cbochs** [comment]
Nice! I think `tag` conveys the same information I was hoping for with `mark`. The API would remain pretty similar, where you have:

```lua
require("portal.tag").tag() -- replaces #mark
require("portal.tag").untag() -- replaces #unmark
require("portal.tag").toggle()
...
```

**Comment by cbochs** [comment]
Addressed in ad553803b6a8cc26f1f916d010d6177fad766620
